setup.md

```diff
@@ -233,11 +233,18 @@ Please skip this section if you are not going to use ESMValTool on your
 local machine and go [here](#github-account-advanced).
 
 If you are planning on running ESMValTool on your own machine, please make sure
-that you are able to download CMIP data and that you have several GB of space
+that you are able to download CMIP data and that you have a few GB of space
 available to install conda and ESMValTool, but also enough to make a copy of some
-data (~2Gb).
+data (~125MB) needed for this tutorial.
 
-To download the data required by this tutorial, run the following command using
+You can use ESMValTool to automatically download data needed for test recipes. 
+Please see the [Configuration][lesson-configuration] episode 
+or the [configuration file documentation][config-file] for more information.
+This the recommended option as it has the advantage that data is 
+stored in subdirectories, and features  such as wildcards and recording 
```

Comment:
There is an extra whitespace between "features" and "such".

Reply:
Fixed.